vllm/forward_context.py

```diff
@@ -28,13 +28,13 @@
 @dataclass
 class ForwardContext:
     # copy from vllm_config.compilation_config.static_forward_context
-    attn_layers: Dict[str, Any]
```

Comment:
let's have another PR to create `attn_layers` and `moe_layers` ?

Reply:
or we can use `no_compile_layers`, since they are the layers invisbile to `torch.compile` .

Reply:
Going with `no_compile_layers` for now